example/izhikevich2007_network.nml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0"?>
```

Comment:
Sadly, Arbor has no support for Izhikevich type neurons, a different example with either IAF or Cable types would be better.

Reply:
Can we keep it anyway? It's a concise demo of a network, and it has it's own tutorial on the nml docs: https://docs.neuroml.org/Userdocs/IzhikevichNetworkExample.html

Reply:
Sure, but it won't run in Arbor as of now.
You could fudge it and replace Iz -> IaF with the proper params.

Reply:
I'll move it to the network branch anyway.